code/modules/admin/admin_verbs.dm

```diff
@@ -794,6 +797,18 @@ var/list/admin_verbs_cciaa = list(
 			verbs |= /client/proc/readmin_self
 	feedback_add_details("admin_verb","DAS") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/toggle_aooc()
+	set name = "Toggle AOOC"
+	set category = "Admin"
+	if(holder)
+		if (tao() == 0)
+			to_chat(src, "<span class='interface'>AOOC is now muted</span>")
```

Comment:
```suggestion
			to_chat(src, "<span class='interface'>AOOC is now muted.</span>")
```

Reply:
I also don't think this span class exists. Make it `'notice'`.

Reply:
Changes the span class just makes it blue, which every other preference is black, which is how it is with the interface spanclass. I can still change it if requested, but it wouldn't match the rest.

Reply:
Well yeah, the idea is you make it blue. Interface is a nonexistant span class is the point.